lib/email-client.js

```diff
@@ -6,7 +6,7 @@ const emailWaitMS = config.get( 'emailWaitMS' );
 export default class EmailClient {
 	constructor( mailboxId ) {
 		const apiKey = config.get( 'mailosaurAPIKey' );
-		const Mailosaur = require( 'mailosaur' )( apiKey, 'https://www.mailosaur.com/api/' );
+		const Mailosaur = require( 'mailosaur' )( apiKey, 'https://mailosaur.com/api/' );
```

Comment:
Why this change necessary? Wasn't it working before?

Reply:
Surprisingly, delete requests returns a 301 redirect and thus fail for `https://www.mailosaur.com/api/` (they were not used in the code until now). Using `https://www.mailosaur.com/api/` it works fine.